content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -1,162 +1,144 @@
 ---
-title: Concepts Underlying the Cloud Controller Manager
+title: Cloud Controller Manager
 content_template: templates/concept
 weight: 40
 ---
 
 {{% capture overview %}}
 
-The cloud controller manager (CCM) concept (not to be confused with the binary) was originally created to allow cloud specific vendor code and the Kubernetes core to evolve independent of one another. The cloud controller manager runs alongside other master components such as the Kubernetes controller manager, the API server, and scheduler. It can also be started as a Kubernetes addon, in which case it runs on top of Kubernetes.
+{{< feature-state state="beta" for_k8s_version="v1.11" >}}
 
-The cloud controller manager's design is based on a plugin mechanism that allows new cloud providers to integrate with Kubernetes easily by using plugins. There are plans in place for on-boarding new cloud providers on Kubernetes and for migrating cloud providers from the old model to the new CCM model.
+Cloud infrastructure technologies let you run Kubernetes on public, private, and hybrid clouds.
+Kubernetes believes in automated, API-driven infrastructure without tight coupling between
```

Comment:
@sftim, I read through the changes. The diagram looks nice.
Should the diagram names be, kube-controller-manager and cloud-controller-manager (missing hyphens)?
Should the names be, Kubernetes control plane, Kubernetes nodes?

I think the concept page could use the original Plugin section (instead of placing the content in the What's Next). Something like, `Cloud Controller Manager plugin interface`.
The `What's next` sections could list the two links to the task pages.

Since there are two CCM tasks, what do you think about creating a new task directory off the task root or under one of the existing sub-directories?
Lines 11-12 Seems somewhat unrelated or too general. What is CCM (a daemon similar to kcm that provides an interface between a cloud provider and the Kubernetes core)?
Line 15, I am not sure if the short glossary definition works here. It sounds awkward to say that the CCM `concept`
(lets you run a cluster ...).
The control plane integrates (not interacts)?

Reply:
- I don't plan to fix the diagram in this PR. I hope that that can happen in a separate PR.
- Thanks for the suggestions about the plugin interface and the What's Next section.
- I'll have a think about making a new task directory.
- and I'll bear everything else in mind too.